junit-jupiter-params/src/test/java/org/junit/jupiter/params/provider/MockCsvAnnotationBuilder.java

```diff
@@ -162,10 +196,31 @@ CsvFileSource build() {
 			when(annotation.encoding()).thenReturn(this.encoding);
 			when(annotation.lineSeparator()).thenReturn(this.lineSeparator);
 			when(annotation.numLinesToSkip()).thenReturn(this.numLinesToSkip);
+			when(annotation.ignoreTrailingAndLeadingWhitespace()).thenReturn(ignoreTrailingAndLeadingWhitespace);
 
 			return annotation;
 		}
 
+		CsvFileSource build(boolean ignoreTrailingAndLeadingWhitespace) {
```

Comment:
You shouldn't have to add this method (similar to the comment regarding `@CsvSource` above).